geth/account/importer_test.go

```diff
@@ -0,0 +1,80 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/extkeys"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestManager_ImportExtendedKey_Success(t *testing.T) {
+	extKey := &extkeys.ExtendedKey{}
+	password := "123"
+	acc := accounts.Account{Address: ethcommon.Address{}}
+	key := keystore.Key{
+		PrivateKey: &ecdsa.PrivateKey{
+			PublicKey: ecdsa.PublicKey{},
+		},
+	}
+
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	keystoreMock := NewMockaccountKeyStorer(ctrl)
+	keystoreMock.EXPECT().ImportExtendedKey(extKey, password).
```

Comment:
Where does this `EXPECT()` come from?

Reply:
@themue It's generic method from gomock mocks.

Reply:
Thanks. Looks interesting so totally upper-case.